pierky/arouteserver/config/general.py

```diff
@@ -123,6 +125,9 @@ def get_schema():
             ("strict", "same-as", "authorized_addresses"),
             default="strict"
         )
+        f["next_hop"]["authorized_addresses_list"] = ValidatorListOf(
```

Comment:
Now we can define authorized_addresses_list in general.yml.  
It's nessesary for ipv6 link local address.

Because there two attribute`nexthop` and `nexthop(link-local)` in bgp message:
![image](https://user-images.githubusercontent.com/73118488/160974945-018e1801-832b-4fbc-a320-d530ed4c9f01.png)

But we can only get `nexthop` attribute from bird's filter.  
If the session is established over ipv6 link local address, the `nexthop` attribute will be `::` so that all routes will be rejected.
![image](https://user-images.githubusercontent.com/73118488/160975355-7c8ddecd-1188-4ae6-a19d-a234f455430d.png)
And there is no way to get the `nexthop(link-local)` attribute from bird 2.0.9 now.

So we have to use this policy for all members in the IX: 
![image](https://user-images.githubusercontent.com/73118488/160975109-dd10cc7a-b1ef-44f5-84e9-13c16e88ccd3.png)